### PR TITLE
Use redux in react component

### PR DESCRIPTION
**Pull request by wacoo**
- Add input fields in component
- Update add and remove book functionalities based on the instruction.
- Fix linter errors

### CODE REVIEW

File: src/components/Book.jsx [hunk 1]
```diff
@@ -9,14 +9,14 @@ const Book = () => {
   const dispatch = useDispatch();
```
Comment (t-yanick) [inline src/components/Book.jsx:10]: - [ ] As per project requirements, you should Wrap `App` your with the component from react-redux and not with the RouterProvider . Please check your `App.jsx` file and fix this.
```jsx
  <Provider store={store}>
    <App />
  </Provider>,
```

Comment (t-yanick) [inline src/components/Book.jsx:10]: - [ ] Redux store should be passed as per project requirements i.e. Pass your configured store into the <Provider>
```
  <Provider store={store}>
    <App />
  </Provider>,
```

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @wacoo , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Initial state added to bookSlice
:heavy_check_mark: Books accessed with useSelector
:heavy_check_mark: components for adding and removing books added

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Book.jsx (hunk 1 above)
Comment (wacoo) [reply src/components/Book.jsx:10]: Please check the index.js. You will find that `<App />` is already wrapped in `<Provider store={store}>...</Provider>`. [Here](https://github.com/wacoo/react-book-store/blob/630d9877180a88ef3679d9f80dbf23140375d785/src/index.js#L11C5-L13C16) is a link to the code.

Comment (wacoo) [reply src/components/Book.jsx:10]: Yes, that's what I did. Did you check the link above? [Here](https://github.com/wacoo/react-book-store/blob/630d9877180a88ef3679d9f80dbf23140375d785/src/index.js#L11C5-L13C16)

**Review by iambenkis — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

✔️ Initial state added to bookSlice
✔️ Books accessed with useSelector
✔️ components for adding and removing books added

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @iambenkis in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**